### PR TITLE
[RDY] Add parser for known mutations and attach it to the GUI. 

### PR DESCRIPTION
**Pull request by pvanagtmaal**
~~Tests need to be added.~~
~~Comes after #158~~

### CODE REVIEW

File: dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import nl.tudelft.dnainator.annotation.DRMutation;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+/**
+ * An interface for parsing drug resistant mutations entries, using an Iterator pattern.
+ */
+public interface DRMutationParser {
```
Comment (Unia) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java]: `extends Iterator`?

**Comment by Unia** [comment]
Very nice! :) Next reviewer may merge

File: dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java (hunk 1 above)
Comment (pvanagtmaal) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java]: possibly, however, parsers like the `EdgeParser` follow a similar pattern. I did the same as we did there.

Comment (Unia) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java]: Lets leave it as-is then

**Comment by Unia** [comment]
Still nice :)

File: dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jBatchBuilder.java [hunk 2]
```diff
@@ -208,16 +190,13 @@ private long createAnnotation(Annotation a) {
...
-	}
 
-	private long createDRMutation(DRMutation dr) {
-		mutationProperties.put(DRMutationProperties.ID.name(), dr.getGeneName());
-		mutationProperties.put(DRMutationProperties.TYPE.name(), dr.getType());
-		mutationProperties.put(DRMutationProperties.CHANGE.name(), dr.getChange());
-		mutationProperties.put(DRMutationProperties.FILTER.name(), dr.getFilter());
-		mutationProperties.put(DRMutationProperties.POSITION.name(), dr.getPosition());
-		return batchInserter.createNode(mutationProperties, NodeLabels.DRMUTATION);
+		if (a instanceof DRMutation) {
```
Comment (pvanagtmaal) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jBatchBuilder.java]: Better to use `isMutation()`.

**Comment by ohuang12** [comment]
LGTM, merging after Travis.

**Pull request merged**